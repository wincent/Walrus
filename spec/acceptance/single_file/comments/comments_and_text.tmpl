This file contains comments ## like this one!
which should not appear in the template output...
## This line should be completely missing from the output
## and this one too, because the comments are the first
## (and only) thing on the lines
In the final output, this should be the third line...
