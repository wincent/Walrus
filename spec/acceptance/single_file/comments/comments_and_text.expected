This file contains comments 
which should not appear in the template output...
In the final output, this should be the third line...